server/core/templates/apps.html

```diff
@@ -0,0 +1,10 @@
+(% extends "base_htmx.html" %)
+
+(% block title %)Error(% endblock %)
+
+(% block head %)
+(% endblock %)
+
+(% block body %)
+	(( apps_partial|safe ))
```

Comment:
we shouldn't be rendering the whole body here as a `apps_partial|safe`, surely - just the tags for the list of them? this feels... risky

Reply:
To my understanding `apps_partial` should be responsible for safely rendering its app items, and we have to assume it did this correctly otherwise we cannot include html from other dynamic templates

without safe askama escapes all html tags:
![image](https://github.com/kanidm/kanidm/assets/32853531/86504889-81ad-4d51-a512-1a197db84e90)

Reply:
Since the `apps_partial` is just another rendered template it should extend the parent template, rather than being yeeted in like this.